docs/install.rst

```diff
@@ -45,6 +45,16 @@ This will install the Kanister controller in the ``kanister`` namespace
                                        --namespace kanister
 
 
+Kanister Custom Resource Definitions (CRD)
+==========================================
+
+`Custom Resource Definitions <https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/>`_
+that Kanister needs are, by default, managed by Kanister controller. If Kanister
+controller doesn't have the authorization to create or update CRDs, you can disable
+that using ``controller.updateCRDs`` helm flag. If set to ``false``, helm will
+manage the CRDs instead of Kanister controller.
```

Comment:
```suggestion
`Custom Resource Definitions <https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/>`_
that Kanister needs are, by default, managed by Kanister controller. If the Kanister
controller doesn't have the authorization to create or update CRDs, you can disable
that using the ``controller.updateCRDs`` helm flag. If set to ``false``, helm will
manage the CRDs instead of the Kanister controller.
```

[nit]

Reply:
I think we can add some example use cases on why the controller won't have the RBAC permissions to manage CRDs.

```suggestion
Kanister defines a few `Custom Resource Definitions (CRDs) <https://docs.kanister.io/architecture.html#custom-resources>` to express operation blueprints and action tasks. By default, these CRDs are managed by the Kanister controller. 

In deployment workflows where the Kanister's YAML manifests must be version-controlled, one can disable these auto-managed CRDs, by setting the Helm ``controller.updateCRDs`` flag to ``false``. This causes the Kanister controller to not output the CRDs YAML during installation. Users can then use commands like ``helm template --set controller.updateCRDS=false`` to output the static YAML of the CRDs and the Kanister application.

Note that use this flag on existing CRDs during an upgrade is not supported.
```

Reply:
Hi @ihcsim 
`Kanister's YAML manifests` this is a bit confusing, what do we mean by Kanister's YAML do we mean YAML of CRDs that Kanister uses. Also, that was not the main use cases why we decided to make have this option. The main use case was that they were not willing to run kanister with authz to manage CRDs. 
`the Kanister controller to not output the CRDs YAML during installation` this sentence is also a bit confusing, Kanister controller doesn't actually output CRDs, it either creates or updates them directly.